### ISSUE TITLE
can't for the life of me get a fluid formation plane to do anything

### ISSUE THREAD

**Issue by HueboWasTaken** [AppliedEnergistics/Applied-Energistics-2#5344]
the fluid formation plane refuses to place fluids 
**Describe the bug**
the fluid formation plane won't place fluids in the world when fluids are being imported

**To Reproduce**
place a fluid formation plane, me controller, me drive with a fluid storage cell in it, and a tank with a fluid import bus connected to it. 
make sure the plane is set up to place that fluid
add fluid to the tank
watch as the fluid goes straight into the storage cell while the fluid formation plane does nothing

**Expected behavior**
fluid formation plane places the fluid in the world

**Additional context**
the fluid formation plane is set to the highest priority and the me drive is set to the lowest priority. the fluid formation plane functions fine when a fluid annihilation plane is used to get the fluid instead of importing it from a tank

**Environment**
playing all of fabric 3
screenshot of setup: https://imgur.com/vKEOZnN

- Environment:

**Comment by mindforger** [comment]
this is how the normal formation planes are working so i suggested it works like designed!

**Comment by have-a-boy** [comment]
> this is how the normal formation planes are working so i suggested it works like designed!

This isn't true. The behavior of a _normal_ formation plane in block placement mode is consistent. An item inserted into the network will be "stored" according to priority by the formation plane, regardless of whether it was imported by a bus, piped into an interface or inserted via terminal. I've recreated OPs setup both with an item and a fluid variant:

### Same setup as the OP (fluid import bus, fluid formation plane and fluid storage disk in ME drive)
- Fluid inserted by a player via fluid terminal is placed by the fluid formation plane, according to priority.
- Fluid imported by the fluid import bus is first stored in the storage disk, disregarding the priorities.

### Same setup as the OP but disk removed from ME drive
- Fluid imported by the fluid import bus is placed by the fluid formation plane, if it is the only valid destination. Which means the path from fluid import bus to fluid formation plane works in principle.

### Same setup as the OP but with normal item equivalents
- Item inserted by a player via terminal is placed by the formation plane, according to priority.
- Item imported by the import bus is placed by the formation plane, according to priority.

In conclusion, it seems only the fluid formation plane is inconsistent regarding the storage priority.

**Comment by mindforger** [comment]
oh well yeah now after re-reading it again i noticed i have misread the additional context ... sorry

>the fluid formation plane is set to the highest priority and the me drive is set to the lowest priority. the fluid formation plane functions fine when a fluid annihilation plane is used to get the fluid instead of importing it from a tank

i did misread that as WHEN the planes are setup like described they work fine 

again my fault sorry

**Comment by Technici4n** [comment]
Pretty sure this was fixed for 1.17 in #5541. Please reopen if the issue persists with 1.17.
